src/test/scala/edu/ie3/osmogrid/guardian/run/SubGridHandlingSpec.scala

```diff
@@ -113,13 +113,18 @@ class SubGridHandlingSpec
     "process and update lv results correctly" in {
       val lv = mockSubGrid(1, MV_10KV, LV)
 
-      val commonNode = lv.getRawGrid.getNodes.asScala
-        .filter(_.isSlack)
+      val mvNode = lv.getRawGrid.getTransformer2Ws.asScala
         .toSeq(0)
+        .getNodeA
         .copy()
         .slack(false)
         .build()
 
+      val commonNode = mvNode
+        .copy()
+        .subnet(3)
+        .build()
```

Comment:
Can be simplified imho

```suggestion
      val commonNode = lv.getRawGrid.getTransformer2Ws.asScala
        .toSeq(0)
        .getNodeA
        .copy()
        .slack(false)
        .subnet(3)
        .build()
```